localflavor/br/models.py

```diff
@@ -18,3 +22,48 @@ def deconstruct(self):
         name, path, args, kwargs = super(BRStateField, self).deconstruct()
         del kwargs['choices']
         return name, path, args, kwargs
+
+
+class BRCPFField(CharField):
+    """
+    A model field for the brazilian document named of CPF (Cadastro de Pessoa Física)
+
+    .. versionadded:: 2.1
+    """
+
+    description = _("CPF Document")
+
+    def __init__(self, *args, **kwargs):
+        kwargs['max_length'] = 14
+        super(BRCPFField, self).__init__(*args, **kwargs)
+        self.validators.append(validators.BRCPFValidator())
+
+
+class BRCNPJField(CharField):
+    """
+    A model field for the brazilian document named of CNPJ (Cadastro Nacional de Pessoa Jurídica)
+
+    .. versionadded:: 2.1
+    """
+
+    description = _("CNPJ Document")
+
+    def __init__(self, *args, **kwargs):
+        kwargs['max_length'] = 18
+        super(BRCNPJField, self).__init__(*args, **kwargs)
+        self.validators.append(validators.BRPostalCodeValidator())
```

Comment:
I think this should be the `BRCNPJValidator`, not the `BRPostalCodeValidator`. I'm a bit surprised that the tests didn't catch this.